--- Atspi-2.0.gir-19.10	2020-07-05 17:31:35.244205276 +0200
+++ Atspi-2.0.gir-20.04	2020-07-05 17:30:41.588798097 +0200
@@ -6,7 +6,7 @@
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
             xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
-  <include name="Gio" version="2.0"/>         
+  <include name="DBus" version="1.0"/>
   <include name="GLib" version="2.0"/>
   <include name="GObject" version="2.0"/>
   <package name="atspi-2"/>
@@ -1686,7 +1686,7 @@
         <type name="utf8" c:type="char*"/>
       </field>
       <field name="bus">
-        <type name="Gio.DBusConnection" c:type="GDBusConnection*"/>
+        <type name="DBus.Connection" c:type="DBusConnection*"/>
       </field>
       <field name="root">
         <type name="gpointer" c:type="_AtspiAccessible*"/>
@@ -2861,181 +2861,181 @@
     <constant name="DBUS_INTERFACE_ACCESSIBLE"
               value="org.a11y.atspi.Accessible"
               c:type="ATSPI_DBUS_INTERFACE_ACCESSIBLE">
-      <source-position filename="../atspi/atspi-constants.h" line="1453"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1467"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_ACTION"
               value="org.a11y.atspi.Action"
               c:type="ATSPI_DBUS_INTERFACE_ACTION">
-      <source-position filename="../atspi/atspi-constants.h" line="1454"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1468"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_APPLICATION"
               value="org.a11y.atspi.Application"
               c:type="ATSPI_DBUS_INTERFACE_APPLICATION">
-      <source-position filename="../atspi/atspi-constants.h" line="1455"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1469"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_CACHE"
               value="org.a11y.atspi.Cache"
               c:type="ATSPI_DBUS_INTERFACE_CACHE">
-      <source-position filename="../atspi/atspi-constants.h" line="1452"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1466"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_COLLECTION"
               value="org.a11y.atspi.Collection"
               c:type="ATSPI_DBUS_INTERFACE_COLLECTION">
-      <source-position filename="../atspi/atspi-constants.h" line="1456"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1470"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_COMPONENT"
               value="org.a11y.atspi.Component"
               c:type="ATSPI_DBUS_INTERFACE_COMPONENT">
-      <source-position filename="../atspi/atspi-constants.h" line="1457"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1471"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_DEC"
               value="org.a11y.atspi.DeviceEventController"
               c:type="ATSPI_DBUS_INTERFACE_DEC">
-      <source-position filename="../atspi/atspi-constants.h" line="1449"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1463"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_DEVICE_EVENT_LISTENER"
               value="org.a11y.atspi.DeviceEventListener"
               c:type="ATSPI_DBUS_INTERFACE_DEVICE_EVENT_LISTENER">
-      <source-position filename="../atspi/atspi-constants.h" line="1450"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1464"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_DOCUMENT"
               value="org.a11y.atspi.Document"
               c:type="ATSPI_DBUS_INTERFACE_DOCUMENT">
-      <source-position filename="../atspi/atspi-constants.h" line="1458"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1472"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_EDITABLE_TEXT"
               value="org.a11y.atspi.EditableText"
               c:type="ATSPI_DBUS_INTERFACE_EDITABLE_TEXT">
-      <source-position filename="../atspi/atspi-constants.h" line="1459"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1473"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_EVENT_KEYBOARD"
               value="org.a11y.atspi.Event.Keyboard"
               c:type="ATSPI_DBUS_INTERFACE_EVENT_KEYBOARD">
-      <source-position filename="../atspi/atspi-constants.h" line="1460"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1474"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_EVENT_MOUSE"
               value="org.a11y.atspi.Event.Mouse"
               c:type="ATSPI_DBUS_INTERFACE_EVENT_MOUSE">
-      <source-position filename="../atspi/atspi-constants.h" line="1461"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1475"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_EVENT_OBJECT"
               value="org.a11y.atspi.Event.Object"
               c:type="ATSPI_DBUS_INTERFACE_EVENT_OBJECT">
-      <source-position filename="../atspi/atspi-constants.h" line="1462"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1476"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_EVENT_SCREEN_READER"
               value="org.a11y.atspi.Event.ScreenReader"
               c:type="ATSPI_DBUS_INTERFACE_EVENT_SCREEN_READER">
-      <source-position filename="../atspi/atspi-constants.h" line="1474"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1488"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_HYPERLINK"
               value="org.a11y.atspi.Hyperlink"
               c:type="ATSPI_DBUS_INTERFACE_HYPERLINK">
-      <source-position filename="../atspi/atspi-constants.h" line="1463"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1477"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_HYPERTEXT"
               value="org.a11y.atspi.Hypertext"
               c:type="ATSPI_DBUS_INTERFACE_HYPERTEXT">
-      <source-position filename="../atspi/atspi-constants.h" line="1464"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1478"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_IMAGE"
               value="org.a11y.atspi.Image"
               c:type="ATSPI_DBUS_INTERFACE_IMAGE">
-      <source-position filename="../atspi/atspi-constants.h" line="1465"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1479"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_REGISTRY"
               value="org.a11y.atspi.Registry"
               c:type="ATSPI_DBUS_INTERFACE_REGISTRY">
-      <source-position filename="../atspi/atspi-constants.h" line="1443"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1457"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_SELECTION"
               value="org.a11y.atspi.Selection"
               c:type="ATSPI_DBUS_INTERFACE_SELECTION">
-      <source-position filename="../atspi/atspi-constants.h" line="1466"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1480"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_SOCKET"
               value="org.a11y.atspi.Socket"
               c:type="ATSPI_DBUS_INTERFACE_SOCKET">
-      <source-position filename="../atspi/atspi-constants.h" line="1471"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1485"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_TABLE"
               value="org.a11y.atspi.Table"
               c:type="ATSPI_DBUS_INTERFACE_TABLE">
-      <source-position filename="../atspi/atspi-constants.h" line="1467"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1481"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_TABLE_CELL"
               value="org.a11y.atspi.TableCell"
               c:type="ATSPI_DBUS_INTERFACE_TABLE_CELL">
-      <source-position filename="../atspi/atspi-constants.h" line="1468"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1482"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_TEXT"
               value="org.a11y.atspi.Text"
               c:type="ATSPI_DBUS_INTERFACE_TEXT">
-      <source-position filename="../atspi/atspi-constants.h" line="1469"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1483"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_INTERFACE_VALUE"
               value="org.a11y.atspi.Value"
               c:type="ATSPI_DBUS_INTERFACE_VALUE">
-      <source-position filename="../atspi/atspi-constants.h" line="1470"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1484"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_NAME_REGISTRY"
               value="org.a11y.atspi.Registry"
               c:type="ATSPI_DBUS_NAME_REGISTRY">
-      <source-position filename="../atspi/atspi-constants.h" line="1441"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1455"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_PATH_DEC"
               value="/org/a11y/atspi/registry/deviceeventcontroller"
               c:type="ATSPI_DBUS_PATH_DEC">
-      <source-position filename="../atspi/atspi-constants.h" line="1448"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1462"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_PATH_NULL"
               value="/org/a11y/atspi/null"
               c:type="ATSPI_DBUS_PATH_NULL">
-      <source-position filename="../atspi/atspi-constants.h" line="1445"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1459"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_PATH_REGISTRY"
               value="/org/a11y/atspi/registry"
               c:type="ATSPI_DBUS_PATH_REGISTRY">
-      <source-position filename="../atspi/atspi-constants.h" line="1442"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1456"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_PATH_ROOT"
               value="/org/a11y/atspi/accessible/root"
               c:type="ATSPI_DBUS_PATH_ROOT">
-      <source-position filename="../atspi/atspi-constants.h" line="1446"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1460"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <constant name="DBUS_PATH_SCREEN_READER"
               value="/org/a11y/atspi/screenreader"
               c:type="ATSPI_DBUS_PATH_SCREEN_READER">
-      <source-position filename="../atspi/atspi-constants.h" line="1473"/>
+      <source-position filename="../atspi/atspi-constants.h" line="1487"/>
       <type name="utf8" c:type="gchar*"/>
     </constant>
     <function-macro name="DEVICE_LISTENER"
@@ -3214,7 +3214,7 @@
             <type name="DeviceListener" c:type="AtspiDeviceListener*"/>
           </instance-parameter>
           <parameter name="event" transfer-ownership="none">
-            <type name="DeviceEvent" c:type="AtspiDeviceEvent*"/>
+            <type name="DeviceEvent" c:type="const AtspiDeviceEvent*"/>
           </parameter>
         </parameters>
       </virtual-method>
@@ -5929,11 +5929,11 @@
         </parameter>
       </parameters>
     </function-macro>
-    <constant name="ROLE_COUNT" value="128" c:type="ATSPI_ROLE_COUNT">
+    <constant name="ROLE_COUNT" value="130" c:type="ATSPI_ROLE_COUNT">
       <doc xml:space="preserve"
            filename="../atspi/atspi-constants.h"
-           line="1381">One higher than the highest valid value of #AtspiRole.</doc>
-      <source-position filename="../atspi/atspi-constants.h" line="1386"/>
+           line="1395">One higher than the highest valid value of #AtspiRole.</doc>
+      <source-position filename="../atspi/atspi-constants.h" line="1400"/>
       <type name="gint" c:type="gint"/>
     </constant>
     <record name="Range"
@@ -7595,13 +7595,39 @@
 inserted, e.g. in revision history or a content view providing suggestions
 from reviewers. @Since: 2.34.</doc>
       </member>
-      <member name="last_defined"
+      <member name="mark"
               value="127"
+              c:identifier="ATSPI_ROLE_MARK"
+              glib:nick="mark">
+        <doc xml:space="preserve"
+             filename="../atspi/atspi-constants.h"
+             line="1243">A run of content that is marked or highlighted, such as for
+reference purposes, or to call it out as having a special purpose. If the
+marked content has an associated section in the document elaborating on the
+reason for the mark, then %ATSPI_RELATION_DETAILS should be used on the mark
+to point to that associated section. In addition, the reciprocal relation
+%ATSPI_RELATION_DETAILS_FOR should be used on the associated content section
+to point back to the mark. @Since: 2.36.</doc>
+      </member>
+      <member name="suggestion"
+              value="128"
+              c:identifier="ATSPI_ROLE_SUGGESTION"
+              glib:nick="suggestion">
+        <doc xml:space="preserve"
+             filename="../atspi/atspi-constants.h"
+             line="1250">A container for content that is called out as a proposed
+change from the current version of the document, such as by a reviewer of the
+content. This role should include either %ATSPI_ROLE_CONTENT_DELETION and/or
+%ATSPI_ROLE_CONTENT_INSERTION children, in any order, to indicate what the
+actual change is. @Since: 2.36</doc>
+      </member>
+      <member name="last_defined"
+              value="129"
               c:identifier="ATSPI_ROLE_LAST_DEFINED"
               glib:nick="last-defined">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1243">Not a valid role, used for finding end of
+             line="1255">Not a valid role, used for finding end of
  enumeration.</doc>
       </member>
       <function name="get_name" c:identifier="atspi_role_get_name">
@@ -7631,8 +7657,8 @@
               c:type="ATSPI_SCROLLTYPE_COUNT">
       <doc xml:space="preserve"
            filename="../atspi/atspi-constants.h"
-           line="1434">One higher than the highest valid value of #AtspiScrollType.</doc>
-      <source-position filename="../atspi/atspi-constants.h" line="1439"/>
+           line="1448">One higher than the highest valid value of #AtspiScrollType.</doc>
+      <source-position filename="../atspi/atspi-constants.h" line="1453"/>
       <type name="gint" c:type="gint"/>
     </constant>
     <function-macro name="SELECTION"
@@ -7700,7 +7726,7 @@
                  c:type="AtspiScrollType">
       <doc xml:space="preserve"
            filename="../atspi/atspi-constants.h"
-           line="1404">Enumeration used by interface #AtspiAccessible to specify where an
+           line="1418">Enumeration used by interface #AtspiAccessible to specify where an
 #AtspiAccessible object should be placed on the screen when using scroll_to.</doc>
       <member name="top_left"
               value="0"
@@ -7708,7 +7734,7 @@
               glib:nick="top-left">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1406">Scroll the object to the top left corner of the
+             line="1420">Scroll the object to the top left corner of the
 window.</doc>
       </member>
       <member name="bottom_right"
@@ -7717,7 +7743,7 @@
               glib:nick="bottom-right">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1408">Scroll the object to the bottom right corner of
+             line="1422">Scroll the object to the bottom right corner of
 the window.</doc>
       </member>
       <member name="top_edge"
@@ -7726,7 +7752,7 @@
               glib:nick="top-edge">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1410">Scroll the object to the top edge of the window.</doc>
+             line="1424">Scroll the object to the top edge of the window.</doc>
       </member>
       <member name="bottom_edge"
               value="3"
@@ -7734,7 +7760,7 @@
               glib:nick="bottom-edge">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1411">Scroll the object to the bottom edge of the
+             line="1425">Scroll the object to the bottom edge of the
 window.</doc>
       </member>
       <member name="left_edge"
@@ -7743,7 +7769,7 @@
               glib:nick="left-edge">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1413">Scroll the object to the left edge of the
+             line="1427">Scroll the object to the left edge of the
 window.</doc>
       </member>
       <member name="right_edge"
@@ -7752,7 +7778,7 @@
               glib:nick="right-edge">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1415">Scroll the object to the right edge of the
+             line="1429">Scroll the object to the right edge of the
 window.</doc>
       </member>
       <member name="anywhere"
@@ -7761,7 +7787,7 @@
               glib:nick="anywhere">
         <doc xml:space="preserve"
              filename="../atspi/atspi-constants.h"
-             line="1417">Scroll the object to application-dependent position
+             line="1431">Scroll the object to application-dependent position
 on the window.</doc>
       </member>
     </enumeration>
@@ -11343,7 +11369,7 @@
           <doc xml:space="preserve"
                filename="../atspi/atspi-gmain.c"
                line="517">the connection</doc>
-          <type name="Gio.DBusConnection" c:type="GDBusConnection*"/>
+          <type name="DBus.Connection" c:type="DBusConnection*"/>
         </parameter>
         <parameter name="context" transfer-ownership="none">
           <doc xml:space="preserve"
@@ -11375,7 +11401,7 @@
           <doc xml:space="preserve"
                filename="../atspi/atspi-gmain.c"
                line="595">the server</doc>
-          <type name="Gio.DBusServer" c:type="GDBusServer*"/>
+          <type c:type="DBusServer*"/>
         </parameter>
         <parameter name="context" transfer-ownership="none">
           <doc xml:space="preserve"
@@ -11604,7 +11630,7 @@
               introspectable="0">
       <source-position filename="../atspi/atspi-misc.h" line="41"/>
       <return-value transfer-ownership="full">
-        <type name="Gio.DBusConnection" c:type="GDBusConnection*"/>
+        <type name="DBus.Connection" c:type="DBusConnection*"/>
       </return-value>
     </function>
     <function name="get_desktop" c:identifier="atspi_get_desktop">
@@ -11861,7 +11887,7 @@
         <parameter name="accessible" transfer-ownership="none">
           <doc xml:space="preserve"
                filename="../atspi/atspi-registry.c"
-               line="532">the #AtspiAccessible corresponding to the window to select.
+               line="531">the #AtspiAccessible corresponding to the window to select.
              should be a top-level window with a role of
              ATSPI_ROLE_APPLICATION.</doc>
           <type name="Accessible" c:type="AtspiAccessible*"/>
